backend/globaleaks/db/migrations/update_69/__init__.py

```diff
@@ -0,0 +1,324 @@
+            template_mail_config.update_date = datetime_now()
+            self.session_new.add(template_mail_config)
+            self.entries_count['ConfigL10N'] += 1
+
+            template_mail_config = self.model_to['ConfigL10N']()
+            template_mail_config.tid = 1
+            template_mail_config.lang = lan
+            template_mail_config.var_name = 'accreditor_signup_external_organization_alert_mail_title'
+            template_mail_config.value = 'An new external organization has been accredited.'
+            template_mail_config.update_date = datetime_now()
+            self.session_new.add(template_mail_config)
+            self.entries_count['ConfigL10N'] += 1
+
+            template_mail_config = self.model_to['ConfigL10N']()
+            template_mail_config.tid = 1
+            template_mail_config.lang = lan
+            template_mail_config.var_name = 'sign_up_external_organization_mail_title'
+            template_mail_config.value = 'external organization confirm accreditation'
+            template_mail_config.update_date = datetime_now()
+            self.session_new.add(template_mail_config)
+            self.entries_count['ConfigL10N'] += 1
+
+            template_mail_config = self.model_to['ConfigL10N']()
+            template_mail_config.tid = 1
+            template_mail_config.lang = lan
+            template_mail_config.var_name = 'sign_up_external_organization_mail_template'
+            template_mail_config.value = 'Dear {RecipientName},\n\nYour accreditation request is in status: {AccreditationStatus}\n\nClick the link to confirm the request:\n\n{ActivationUrl}\n\nKind regards,\n\n{NodeName}'
+            template_mail_config.update_date = datetime_now()
+            self.session_new.add(template_mail_config)
+            self.entries_count['ConfigL10N'] += 1
+
+            template_mail_config = self.model_to['ConfigL10N']()
+            template_mail_config.tid = 1
+            template_mail_config.lang = lan
+            template_mail_config.var_name = 'sign_up_external_organization_info_mail_title'
+            template_mail_config.value = 'external organization request accreditation'
+            template_mail_config.update_date = datetime_now()
+            self.session_new.add(template_mail_config)
+            self.entries_count['ConfigL10N'] += 1
+
+            template_mail_config = self.model_to['ConfigL10N']()
+            template_mail_config.tid = 1
+            template_mail_config.lang = lan
+            template_mail_config.var_name = 'sign_up_external_organization_info_mail_template'
+            template_mail_config.value = 'Dear {RecipientName},\n\nYour accreditation request is in status: {AccreditationStatus}\n\n{AccreditationMotivationText}\n\nKind regards,\n\n{NodeName}'
+            template_mail_config.update_date = datetime_now()
+            self.session_new.add(template_mail_config)
+            self.entries_count['ConfigL10N'] += 1
+
+    def add_backup_configs(self):
```

Comment:
By configuring correctly the variables in the globaleaks/models this code is not necessary.